librz/util/vector.c

```diff
@@ -236,6 +236,30 @@ RZ_API void *rz_vector_flush(RzVector *vec) {
 	return r;
 }
 
+RZ_API void rz_vector_insertion_sort(RzVector *vec, RzVectorComparator cmp, bool reverse) {
```

Comment:
Insertion sort is pretty much one of the slowest ways to sort. You can just pick the quicksort from `rz_pvector_sort()` and adjust it for non-p vectors.
The new sort function will also need unit tests then.